ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainActivity.kt

```diff
@@ -1,19 +1,39 @@
 package ai.nnstreamer.ml.inference.offloading
 
+import ai.nnstreamer.ml.inference.offloading.ui.theme.NnstreamerandroidTheme
+import android.content.ComponentName
+import android.content.Context
+import android.content.Intent
+import android.content.ServiceConnection
 import android.os.Bundle
+import android.os.IBinder
+import android.view.View
+import android.widget.Button
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Surface
-import androidx.compose.material3.Text
-import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.tooling.preview.Preview
-import ai.nnstreamer.ml.inference.offloading.ui.theme.NnstreamerandroidTheme
-import android.content.Intent
 
 class MainActivity : ComponentActivity() {
+    private val TAG = "MainActivity"
+    private lateinit var mService: MainService
+    private var mBound: Boolean = false
```

Comment:
Q: is it necessary?
if bound, mService in onServiceConnected() is not null.
when disconnected (onServiceDisconnected() or onStop()), mService = null;

then we can handle button event with condition mService is not null, and remove boolean flag mBound.

Reply:
Oh you're right.
I just followed the tutorial without thinking😅
`mBound` seems to be an unnecessary variable.
Thank you!